src/core/ui/components/sign-in/screens/loading.tsx

```diff
@@ -0,0 +1,63 @@
+import { Header } from '../layout/header';
+import { Content, Section } from '../layout/content';
+import { Footer } from '../layout/footer';
+import { Text, Size } from '../../primitives/text';
+import { useLoginMethods, getIconsById } from '../../../context/login-methods';
+import { Flex } from '../../primitives/container';
+import { HighlightedIcon } from '../../primitives/icon';
+import { Loader } from '../../primitives/loader';
+import { BaseButton } from '../../primitives/button';
+import { classNames } from '../../../../../shared/utils/classnames';
+import text from '../../primitives/text.module.css';
+import margin from '../../primitives/margin.module.css';
+import styles from './loading.module.css';
+
+export const LoadingScreen = ({ navigateBack }) => {
+  const { selectedLoginMethod: selected } = useLoginMethods();
+
+  const selectedLoginMethod = selected;
```

Comment:
Gotta remove this

Reply:
is this still a todo for this PR?

Reply:
Sorry, I've just pushed the fix! I will merge this PR shortly